Makefile

```diff
@@ -128,14 +128,17 @@ build-rococo-release:
 build-mainnet-release:
 	cargo build --locked --features  frequency --profile production
 
-PHONY: test
+.PHONY: test
 test:
 	cargo test --workspace --locked --features all-frequency-features
 
 integration-test:
 	./scripts/run_integration_tests.sh
 
-PHONY: version
+# Pull the Polkadot version from the polkadot-cli package and remove the leading v
+POLKADOT_VERSION=$(shell cargo tree --depth 0 --manifest-path ./node/service/Cargo.toml --package polkadot-cli | grep polkadot-cli | cut -d " " -f 2 | tr -d "v")
```

Comment:
This is a bit strong for something that we could make lighter, but it works and is fairly consistent.